pkg/machines/components/aggregateStatusCards.scss

```diff
@@ -23,29 +23,15 @@
   grid-gap: var(--cards-gutter-y) var(--cards-gutter-x);
   grid-template-columns: repeat(auto-fit, minmax(var(--cards-min), 1fr));
 
-  > div {
-    /* No funky negative margins are needed for grid */
```

Comment:
' machines: Properly adapt card CSS to SCSS '
The commit message does not properly describe what is happening here. I will add proper commit message because this needs to stay unsquashed.